### PR TITLE
Update README for 22.09 release

### PR DESCRIPTION
**Pull request by mc-nv**

### CODE REVIEW

**Review by jbkyang-nvi — DISMISSED** [review]

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  algorithm to Model Analyzer to drastically reduce search time.
+
+* Added 
+  [GPU metrics](https://github.com/triton-inference-server/server/blob/main/docs/user_guide/perf_analyzer.md#server-side-prometheus-metrics) 
+  gathering to Perf Analyzer, which is also used by Model Analyzer to improve 
+  accuracy of those metrics.
+
+
+## Known Issues
+
```
Comment (nv-kmcgill53) [inline RELEASE.md:73]: We should document the nullptr issue we investigated. @tanmayv25 Can you provide a relnote for this?

Comment (tanmayv25) [reply RELEASE.md:73]: " In certain rare cases with specific backends, triton server may crash with segmentation fault while exiting. Preliminary analysis show that there might be a race condition in clean up of backend/model/instance state objects. Exact root cause is still unknown."

Comment (tanmayv25) [reply RELEASE.md:73]: @mc-nv  @nv-kmcgill53

Comment (nv-kmcgill53) [reply RELEASE.md:73]: Fine with me

Comment (mc-nv) [reply RELEASE.md:73]: @tanmayv25 fixed

**Review by nv-kmcgill53 — APPROVED** [review]

**Pull request merged**